line-sdk/src/main/res/layout/open_chat_info_fragment.xml

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context=".openchat.ui.OpenChatInfoFragment">
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.linecorp.linesdk.openchat.ui.OpenChatInfoViewModel" />
+    </data>
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/openchat_background">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/background"
+            android:paddingStart="16dp"
+            android:paddingEnd="16dp">
+
+            <EditText
+                android:id="@+id/nameEditText"
+                android:layout_width="0dp"
+                android:layout_height="121dp"
+                android:layout_marginEnd="13dp"
+                android:background="@color/background"
+                android:drawablePadding="18dp"
+                android:gravity="center_vertical"
+                android:hint="@string/openchat_create_room_name_placeholder"
+                android:imeOptions="actionNext"
+                android:includeFontPadding="true"
+                android:maxLength="50"
```

Comment:
To make the value consistent between XML and code, we need to define an `integers.xml` and use the same resource integer.

Create an `integers.xml`,
```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <integer name="max_profile_name_length">20</integer>
    <integer name="max_chatroom_name_length">50</integer>
    <integer name="max_chatroom_description_length">200</integer>
</resources>
```
In `OpenChatInfoViewModel.kt`,
```kotlin
class OpenChatInfoViewModel(application: Application) : AndroidViewModel(application) {
    ...

    val chatroomNameIndicator: LiveData<String> = Transformations.map(chatroomName) { name ->
        createTextMaxLengthIndicatorString(name, getResourceInt(R.integer.max_chatroom_name_length))
    }

    val descriptionNameIndicator: LiveData<String> = Transformations.map(description) { name ->
        createTextMaxLengthIndicatorString(
            name,
            getResourceInt(R.integer.max_chatroom_description_length)
        )
    }
    
    private fun getResourceInt(@IntegerRes resId: Int): Int =
        getApplication<Application>().resources.getInteger(resId)

    ...
}
```
In `open_chat_info_fragment.xml`,
```xml
android:maxLength="@integer/max_chatroom_name_length"
```